osu.Game.Tests/Resources/overlapping-control-points.osu

```diff
@@ -4,16 +4,16 @@ osu file format v14
 
 // Timing then inherited
 0,500,4,2,0,100,1,0
-0,-66.6666666666667,4,3,0,100,0,1
+0,-66.6666666666667,4,3,0,100,0,9
 
 // Inherited then timing (equivalent to previous)
 1000,-66.6666666666667,4,3,0,100,0,1
 1000,500,4,2,0,100,1,0
 
 // Inherited then timing (different to previous)
-2000,-133.333333333333,4,1,0,100,0,0
-2000,250,4,2,0,100,1,0
+2000,-133.333333333333,4,1,0,100,0,8
+2000,250,4,2,0,100,1,8
 
 // Timing then inherited (different to previous)
-3000,500,4,2,0,100,1,0
+3000,500,4,2,0,100,1,8
```

Comment:
I don't think this test should have been changed like this. It's actually hiding a potential flaw in implementation, and makes it very hard to review / logically understand. Best to define a new `.osu` file for new cases like this.